### ISSUE TITLE
total_pix

### ISSUE THREAD

**Issue by Fredthearboro** [AbyszOne/Abysz_lab#3]
Trying to run this on Windows 10. Downloaded all requirements. Got it working for a maximum of ~5 images every time before it crashes with the error  "**total_pix = img.size # obtener el número total de píxeles en la imagen
AttributeError: 'NoneType' object has no attribute 'size'**"

I tried multiple times. 
To get it to work without giving me weird permission errors I had to delete the "source" folder every retry.

Below is a screenshot for more context.
![image](https://github.com/AbyszOne/Abysz_lab/assets/44513681/626af795-2aaa-4f09-95b6-079a794345c6)

**Comment by AbyszOne** [comment]
Frankly it is a puzzle what is your problem. Considering that I have never been told or experienced a similar error in linux or windows, and from the text provided, I can only deduce that it is more related to the logistics of permissions in your system.
The fact that it works for a certain number of images is weirder. Have you tried with a different set of images and in different location folders?

**Comment by AbyszOne** [comment]
I have received another strange inquiry today. Confirm me one thing, are you using the latest update of Automatic1111? I think that may be the origin of the conflict.

**Comment by federicobuilds** [comment]
> I have received another strange inquiry today. Confirm me one thing, are you using the latest update of Automatic1111? I think that may be the origin of the conflict.

Yes, I am up to date on everything. I even use git pull on the command flags to auto update when I run the program 


I also tried different images and folders

**Comment by AbyszOne** [comment]
Thats what I mean. Recent GUI and gradio updates could broke some of the code. I'll test it.
